src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationHandlers.cs

```diff
@@ -1494,13 +1497,16 @@ ProviderTypes.Yandex  or ProviderTypes.Zoom
                 // Zoho returns the username as a custom "Display_Name" node:
                 ProviderTypes.Zoho => (string?) context.UserInfoResponse?["Display_Name"],
 
+                // Genesys returns the username as a custom "Display_Name" node:
+                ProviderTypes.Genesys => context.MergedPrincipal.GetClaim("name"),
```

Comment:
Same remark here, it's not valid and doesn't match the code comment. If the standard `name` claim is used, remove that line.